src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.Terminal.TerminalControl
         String Text { get; };
         String Html { get; };
         String Rtf { get; };
+        Windows.Foundation.IReference<Microsoft.Terminal.Settings.CopyFormat> Formats { get; };

```

Comment:
see you're using an option type here

Reply:
By design. See/Moving discussion to this other thread:
https://github.com/microsoft/terminal/pull/6004#discussion_r458138986